src/js/reducers/limitOrderReducer.js

```diff
@@ -146,14 +146,44 @@ const limitOrder = (state = initState, action) => {
       newState.currentPathIndex += 1
       return newState
     }
-
+    case "LIMIT_ORDER.ADD_LIST_ORDER": {
+      const listOrder = action.payload;
+      newState.listOrder = listOrder;
+      return newState;
+    }
     case "LIMIT_ORDER.ADD_NEW_ORDER":{
       const {order} = action.payload
       var listOrder = newState.listOrder
       listOrder.unshift(order)
       newState.listOrder = listOrder
       return newState
     }
+    case "LIMIT_ORDER.REMOVE_ORDER": {
+      const { order } = action.payload;
+      const listOrder = JSON.parse(JSON.stringify(newState.listOrder));
+      const target = listOrder.filter(item => item.id === order.id);
+      if (target.length > 0) {
+        listOrder.splice(listOrder.indexOf(target[0]), 1);
+      }
+      newState.listOrder = listOrder;
+      return newState;
+    }
+    case "LIMIT_ORDER.CANCEL_ORDER": {
```

Comment:
Its should call to limit order services got cancel order. You should not hard code for status "cancel"

Reply:
Got it, thanks for pointing out.